src/libs/actions/Policy.ts

```diff
@@ -2553,6 +2563,399 @@ function clearCategoryErrors(policyID: string, categoryName: string) {
+            onyxMethod: Onyx.METHOD.MERGE,
+            key: `${ONYXKEYS.COLLECTION.POLICY}${policyID}`,
+            value: {
+                pendingFields: {
+                    areCategoriesEnabled: null,
+                },
+                pendingAction: null,
+            },
+        },
+    ];
+    const failureData: OnyxUpdate[] = [
+        {
+            onyxMethod: Onyx.METHOD.MERGE,
+            key: `${ONYXKEYS.COLLECTION.POLICY}${policyID}`,
+            value: {
+                areCategoriesEnabled: !isEnabled,
+                pendingFields: {
+                    areCategoriesEnabled: null,
+                },
+                pendingAction: null,
+            },
+        },
+    ];
+
+    const parameters: EnablePolicyCategoriesParams = {policyID, isEnabled};
+
+    API.write(WRITE_COMMANDS.ENABLE_POLICY_CATEGORIES, parameters, {
+        optimisticData,
+        successData,
+        failureData,
+    });
+
+    if (isEnabled) {
+        const navigationAction = getIsNarrowLayout() ? Navigation.goBack : () => Navigation.navigate(ROUTES.WORKSPACE_CATEGORIES.getRoute(policyID));
+
+        navigationAction();
+    }
+}
+
+function enablePolicyConnections(policyID: string, isEnabled: boolean) {
+    const optimisticData: OnyxUpdate[] = [
+        {
+            onyxMethod: Onyx.METHOD.MERGE,
+            key: `${ONYXKEYS.COLLECTION.POLICY}${policyID}`,
+            value: {
+                areConnectionsEnabled: isEnabled,
+                pendingFields: {
+                    areConnectionsEnabled: CONST.RED_BRICK_ROAD_PENDING_ACTION.UPDATE,
+                },
+                pendingAction: CONST.RED_BRICK_ROAD_PENDING_ACTION.UPDATE,
+            },
+        },
+    ];
+    const successData: OnyxUpdate[] = [
+        {
+            onyxMethod: Onyx.METHOD.MERGE,
+            key: `${ONYXKEYS.COLLECTION.POLICY}${policyID}`,
+            value: {
+                pendingFields: {
+                    areConnectionsEnabled: null,
+                },
+                pendingAction: null,
+            },
+        },
+    ];
+    const failureData: OnyxUpdate[] = [
+        {
+            onyxMethod: Onyx.METHOD.MERGE,
+            key: `${ONYXKEYS.COLLECTION.POLICY}${policyID}`,
+            value: {
+                areConnectionsEnabled: !isEnabled,
+                pendingFields: {
+                    areConnectionsEnabled: null,
+                },
+                pendingAction: null,
+            },
+        },
+    ];
+
+    const parameters: EnablePolicyConnectionsParams = {policyID, isEnabled};
+
+    API.write(WRITE_COMMANDS.ENABLE_POLICY_CONNECTIONS, parameters, {
+        optimisticData,
+        successData,
+        failureData,
+    });
+
+    // TODO: Integrate navigation to a screen, once it appears
+    // if (isEnabled) {
+    //     const navigationAction = getIsNarrowLayout() ? Navigation.goBack : () => Navigation.navigate(ROUTES.WORKSPACE_.getRoute(policyID));
+
+    //     navigationAction();
+    // }
```

Comment:
I think we should just remove this. We'll implement this whenever we implement the connection feature
```suggestion
```